src/admin/getting_started_checks/design_removeDefaultBannersListingDetailsPage.php

```diff
@@ -50,18 +50,17 @@ public function isComplete()
         if (geoMaster::is('auctions')) {
             $name = 'listing_auction.tpl';
             $custom = file_get_contents(geoTemplate::getFilePath(geoTemplate::MAIN_PAGE, '', $name));
-            if ((strpos($custom, '<a href="#"><img src="{external file=\'images/banners/banner1_160w.jpg\'}" alt="" /></a>') !== false)) {
+            if (strpos($custom, 'sample_300x100.jpg') || strpos($custom, '1and1_300x100.jpg')) {
```

Comment:
Same here, the one being replaced was `images/banners/banner1_160w.jpg` - is that intentional?

Reply:
yep.

Reply:
> if (strpos($custom, 'sample_300x100.jpg') || strpos($custom, '1and1_300x100.jpg')) {
>     return false;
> }
> return true;
> 

I was going to do it that way at first and didn't because....